app/js/components/EscrowList.jsx

```diff
@@ -57,22 +76,37 @@ const EscrowList = (props) => (<Card className="mt-2">
               {getEscrowState(escrow) === escrowStates.expired && escrow.seller === web3.eth.defaultAccount &&
               <Button color="warning" size="sm" block
                       onClick={() => props.cancelEscrow(escrow.escrowId)}>Cancel</Button>}
-
-              {getEscrowState(escrow) !== escrowStates.waiting && escrow.buyer === web3.eth.defaultAccount &&
-              <Rating rating={escrow.rating} rateTransaction={props.rateTransaction}
-                      escrowId={escrow.escrowId}/>}
+              {getEscrowState(escrow) === escrowStates.released && !escrow.arbitration && escrow.buyer === web3.eth.defaultAccount &&
```

Comment:
`getEscrowState(escrow)` could be assigned to a variable

Reply:
+1

What do you think 
```
 {props.escrows.map(escrow => { escrow.stateText = getEscrowStateText(escrow); })
               .map(escrow => (
```

And then instead of calling `getEscrowStateText` in the conditions, i replace it to `escrow.stateText`